### DISCUSSION TITLE
puttingitonplastic

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#13] [Announcements]
# puttingitonplastic

Exploring the challenges and solutions to plastic recycling, focusing on a new tax approach.

https://nonzerosum.games/puttingitonplastic.html?fbclid=IwAR0tgE73d6Un3HZH-2IvdqthuHEIdLSUopWFV4v-gtzNhiHetB6L8zTe2_s

**Comment by andrewtaneglen** [comment]
I see plastics as one of the many 'one-off natural bounties' we are burning through. The true 'sustainable replacement' cost is never considered, we just burn through it and move to the next thing. I see this as the core of colonialism - not displacement of population (as bad as that is), but the one-off harvesting of resources. The colonists are merely labour-fodder for the capitalists back in the capital (The Alien movies provide the basis of this critique).
How about a plastic tax to directly fund investment in the development of alternatives, to become future market leaders! The aim would be to not ban plastics, but to transcend them.

**Reply by jamesstephenbrown** [reply]
I see your point, yes, oil in general fits this bill, plastic is an outgrowth of that one-off bounty (oil) and is bound to the same fate apparently.. made all the more evident by the fact that in the end we burn it in the waste-to-energy process.

I agree that a plastic tax could be used to fund investment in alternatives, I guess I was focussing on funding my pet glass bottles (an existing alternative) project because that's where I started from, but yes, plastic could philanthropise itself out of existence by funding better alternatives that make it redundant.

I totally agree that deterrent taxes are far more preferable than bans and draconian laws - we live in a free society thankfully, which responds to various pressures, all you can hope to do is apply pressure in the direction you'd like us to go, erecting walls is pointless - walls will fall or spring leaks in response to pressure.

The subject of pressure makes me wonder if we're going to run into a problem when we manage to reduce demand for plastic through these methods while also reducing demand for petrol through electric vehicles, that a sort of equilibrium will be reached where oil companies end up with a surplus of cheap oil which then pressures its way back into industry. I guess we can hope that oil companies having to let oil go for cheap makes expensive drilling projects less attractive and eventually... some day...

**Reply by andrewtaneglen** [reply]
The geopolitics of oil points to something like this. As the price of oil goes up, shale oil (fracking) becomes economical, so takes off (in the US). Those with access to 'easier oil', i.e. Saudi Arabia, then pump up production to bring the price back down again, thereby decimating the shale oil industry. The expectation would be that the moment energy is produced more cheaply elsewhere, than extracting and refining petrol in Saudi Arabia (including government subsidies), investors in Saudi Aramco will walk away, leaving them to burn through their not-insignificant bank balance, until we reach the future history of 'The Socialist Revolution of the post-petroleum Arabian Peninsula'.
There will no doubt still be a market for specialist oil-based products, but this should come at a point where we're no longer just burning through ~97 million barrels of it a day.

**Reply by jamesstephenbrown** [reply]
Ah, so higher prices leads to pumped up production, so lower prices should lead to diminished production, here's hoping - how do you know all this stuff about Saudi Arabian oil funders and subsidies..? Your breadth of knowledge never ceases to amaze.

**Reply by andrewtaneglen** [reply]
Gets crazier when you consider Venezuela has access to more oil than Saudi Arabia and much closer to the US, but 'for some reason' can't maintain a stable government. It's as if geopolitical forces don't want another competitor they are unable to control/profit from. Maybe if Saudi Arabia runs out they'll get their turn.

**Reply by jamesstephenbrown** [reply]
He knows too much, it's time guys, you know what to do.
